op-e2e/system_test.go

```diff
@@ -1496,24 +1498,49 @@ func TestChallenge(t *testing.T) {
 	}
 	cancel()
 
-	// set a timeout for security council to validate output
-	ctx, cancel = context.WithTimeout(context.Background(), 10*time.Second)
+	findTransactionId := func() *big.Int {
```

Comment:
I can understand why you named it `findTransactionId` since it's obvious what it does in the for loop underneath, but looking at that function alone, I don't understand what transaction you're trying to find or what output you want.
Also, using the auto-generated code in bindings would eliminate the need to reference the "ValidationRequested" field as a string. 

Considering these two points, I would suggest removing findTransactionId and doing it this way.

line 1470
```go
securityCouncil, err := bindings.NewSecurityCouncil(cfg.L1Deployments.SecurityCouncilProxy, l1Client)
```

```go
if challengeStatus == chal.StatusNone {
    // check validation request tx exists and not executed
    toBlock := latestBlock(t, l1Client)
    iter, _ := securityCouncil.FilterValidationRequested(&bind.FilterOpts{End: &toBlock}, nil)
    iter.Next()
    tx, err := securityCouncil.Transactions(&bind.CallOpts{}, iter.Event.TransactionId)
    iter.Close()
    ....
}
```